wordpress-mobile/WordPress-iOS
# Remove Internal builds leftovers

## Summary
This PR removes the "Release-Internal" configuration and "WordPress Internal" scheme. Those were used for App Center builds that were uploaded during the release cycle. Those builds were disabled by @mokagio in March 2024. The prototype builds that are uploaded to App Center for each PR are still enabled (referred to as the Alpha configuration)--this is just for those release cycle builds. 

Removes: 
- "Release-Internal" configuration from the project
- "WordPress Internal" scheme from the project
- The xcconfig files for the Internal configuration
- Internal version helper methods in the Fastfile
- Internal code signing methods

To test:
- Verify that CI passes
- Included in that, verify that the prototype build has been successfully created

PR submission checklist:


Testing checklist:
